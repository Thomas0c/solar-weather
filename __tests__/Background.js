import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Background from '../lib/js/app/components/background';

test('renders correctly', () => {
  const tree = renderer.create(<Background />).toJSON();
  expect(tree).toMatchSnapshot();
});
