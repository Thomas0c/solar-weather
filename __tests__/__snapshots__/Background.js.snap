// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityLiveRegion={undefined}
  accessibilityTraits={undefined}
  accessibilityViewIsModal={undefined}
  accessible={undefined}
  collapsable={undefined}
  hitSlop={undefined}
  importantForAccessibility={undefined}
  needsOffscreenAlphaCompositing={undefined}
  onAccessibilityTap={undefined}
  onLayout={undefined}
  onMagicTap={undefined}
  pointerEvents={undefined}
  removeClippedSubviews={undefined}
  renderToHardwareTextureAndroid={undefined}
  shouldRasterizeIOS={undefined}
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "#C0C0C0",
      "height": 1334,
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": 750,
    }
  }
  testID={undefined}
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityLiveRegion={undefined}
    accessibilityTraits={undefined}
    accessibilityViewIsModal={undefined}
    accessible={undefined}
    collapsable={undefined}
    hitSlop={undefined}
    importantForAccessibility={undefined}
    needsOffscreenAlphaCompositing={undefined}
    onAccessibilityTap={undefined}
    onLayout={undefined}
    onMagicTap={undefined}
    pointerEvents={undefined}
    removeClippedSubviews={undefined}
    renderToHardwareTextureAndroid={undefined}
    shouldRasterizeIOS={undefined}
    style={
      Object {
        "backgroundColor": "#C1C7C9",
        "height": "100.00%",
        "width": "100.00%",
      }
    }
    testID={undefined}
  />
</View>
`;
